docs/dev/table/connect.md

```diff
@@ -656,9 +656,11 @@ connector:
 
 The file system connector itself is included in Flink and does not require an additional dependency. A corresponding format needs to be specified for reading and writing rows from and to a file system.
 
+The file system connector can also be defined with a *CREATE TABLE DDL* statement, please see the dedicated page [DDL](ddl.html) examples.
```

Comment:
The purpose of FLINK-13362 is aiming to add documentation about how to create filesystem, kafka, ES connectors by DDL. It would be great if we can add an additional tab called "DDL" besides "Java/Scala" and "YAML", and describe all the supported properties.

Reply:
Actually we still execute DDLs with  Java, Scala or Python, i suggest to move it in the DDL page and make a link here.

Reply:
I mean we need to describe all the supported property keys and values in different connectors.  And I think `connect.md` is a good place to go instead of to list all the connectors again in `ddl.md`.